### DISCUSSION TITLE
Explanation of vulnerability fields appearing in the report

### DISCUSSION THREAD

**Discussion by Mettthew** [security-companion/w3af#641] [General]
hello everyone,

I wanted to ask if by any chance there was any documentation where all the possible fields that appear when searching for a specific vulnerability found through the endpoint `"http://127.0.0.1:5000/scans/scan_id/kb/id"` are explained one by one.

- for example, I know that the `cwe_ids` and `wasc_ids` fields refer to the `Common Weakness Enumeration` and `Web Application Security Consortium`, respectively, while `VulnDB_id` I imagine refers to a Vulnerabilities Database but I don't understand which one specifically, because searching the Web one can find several.
- Another question that comes to mind is what `Uniq_id` refers to.
- and lastly is ` owasp_top_10_references` valid to consider or have they been updated? because I've noticed that the `Owasp version` that appears is `2013` and the link that is generally associated with it doesn't actually lead to what it should indicate, for example: `"owasp_top_10_references":[{"link":"https://www.owasp.org/index.php/Top_10_2013-A1","owasp_version":"2013","risk_id":1}]`

There are also other fields that I don't quite understand, but these are the main ones for now.

**Comment by codders** [comment]
Hi @Mettthew ,

Sorry for the delay in replying here. I've tried to add some detail in the issue that you opened (#643).
